### PR TITLE
Ft home screen

### PR DESCRIPTION
**Pull request by emmanuelkech**

### CODE REVIEW

File: app/src/main/java/com/example/emmy_ibook/adapter/BestReadsAdapter.kt [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import androidx.recyclerview.widget.RecyclerView
+import com.example.emmy_ibook.R
+import com.example.emmy_ibook.fragments.HomeFragment
+import com.example.emmy_ibook.model.BestReads
+
+class BestReadsAdapter(private val context: HomeFragment, private val dataSet: List<BestReads> ) : RecyclerView.Adapter<BestReadsAdapter.BestReadsViewHolder>(){
+
+
+    class BestReadsViewHolder(view: View) : RecyclerView.ViewHolder(view){
+        val bookImage : ImageView = view.findViewById(R.id.br_img)
```
Comment (Ikey22) [inline app/src/main/java/com/example/emmy_ibook/adapter/BestReadsAdapter.kt]: why are you not using viewBinding?

**Comment by emmanuelkech** [comment]
oh no…..i was meant to go back and change it….I switched to findById when I
was trying to test an irregularity.

I’ll review it now.

File: app/src/main/java/com/example/emmy_ibook/adapter/BestReadsAdapter.kt (hunk 1 above)
Comment (emmanuelkech) [reply app/src/main/java/com/example/emmy_ibook/adapter/BestReadsAdapter.kt]: oh no…..I was meant to go back and change it….I switched to findById when I
was trying to test an irregularity.

I’ll review it now.

Comment (Ikey22) [reply app/src/main/java/com/example/emmy_ibook/adapter/BestReadsAdapter.kt]: Finally, you should request to merge to dev and not main

Comment (Ikey22) [reply app/src/main/java/com/example/emmy_ibook/adapter/BestReadsAdapter.kt]: Also, you didn't attach images of the screens

**Pull request merged**